config/configStruct/config.go

```diff
@@ -0,0 +1,57 @@
+package configStruct
+
+import (
+	"github.com/TremblingV5/DouTok/pkg/dtviper"
+)
+
+type Config struct {
```

Comment:
Not each service need all config struct, such as MySQL, Redis HBase is not necessary for some services.